projectfiles/.gitignore

```diff
@@ -0,0 +1,3 @@
+/Xcode-iOS
+/*.xcworkspace
+*.xcworkspacedata
```

Comment:
I'm afraid root .gitignore doesn't cover these 8(

Though, this mostly slipped in. Some of these are really only for my local environment. Will fix.

The XCode project is a good question. I would, but it alone takes about 1G on the drive, given it contains sources of all the dependencies, and binaries of the most prominent: Boost and glib.
I do commit it though, here: https://github.com/singalen/wesnoth_ios/. Check out into projectfiles.
I can add the project as a git submodule, but if mainline ever adds submodules, the one who recursively clones master, will pull the said 1G of files. Plus, iOS project already has submodules of its own, said Boost and glib.

I think the best way would be to add a readme to projectfiles/XCode-iOS, with a build instruction. What do you think?